server/models.py

```diff
@@ -404,7 +405,24 @@ class Batch(BaseModel):
         order_by="BatchResultTallySheet.created_at",
     )
 
-    __table_args__ = (UniqueConstraint("jurisdiction_id", "tabulator", "name"),)
+    # Only one of the following should be populated, enforced via a check constraint defined below
+    last_edited_by_support_user_email = Column(String(200))
+    last_edited_by_jurisdiction_admin_email = Column(String(200))
+    last_edited_by_tally_entry_user_id = Column(
+        String(200), ForeignKey("tally_entry_user.id", ondelete="set null")
+    )
+
+    last_edited_by_tally_entry_user = relationship("TallyEntryUser")
```

Comment:
My understanding of SQL Alchemy relationships is very rudimentary. It's not clear to me what other params I'd want to specify here (e.g. I see that every other relationship specifies `back_populates`)

Reply:
This looks fine!

`back_populates` is only needed if you have a corresponding relationship defined in the other direction as well. E.g. in this case if you wanted a relationship like `TallyEntryUser.batch_tally_sheets_that_i_last_edited`

Reply:
(I left `back_populates` empty because I don't envision any cross-editing needs, e.g. I don't think we should be modifying the tally entry user through `last_edited_by_tally_entry_user`. But I might not be thinking about this in the right way)

Reply:
Got it - thanks for clarifying!